### PR TITLE
Quality assessment update

### LINKED ISSUE #368: Supporting Document v1.1 Section 2.3.2 FIA_MBE_EXT.2
Supporting Document v1.1 Section 2.3.2 FIA_MBE_EXT.2 - This test requires some product, other than the one being tested, to be developed for the purpose of testing. The access required for the testing seems rather invasive and I wonder if it would always be possible to get access to required data in what might be separate and secure internal operating environments. This possibility should be addressed in some manner.
It is assumed that the developer provides the quality assessment report (i.e., it is not created by CC evaluators).  This seems like expert subject matter and it is not clear CC evaluator would or even should be generally qualified to assess that document ala the AGD_OPE/ADV_FSP activity in section 2.3.2.5.1.  Rather the guidance for its evaluation should be broken down and itemized so that it is clear specifically what to look for and what would be good enough, otherwise consistency cannot be assured in any real degree.  Same comment for Section 2.3.4 FIA_MBV_EXT.2.

### LINKED ISSUE #369: Supporting Document v1.1 Section 2.3.3 FIA_MBV_EXT.1
Supporting Document v1.1 Section 2.3.3 FIA_MBV_EXT.1 - Like the quality assessment report , it seems like the performance report should require specialized expertise to evaluate and it is not clear that should be in the domain of CC evaluators in general.  Like other modern CC packages, the activities and expectations should be broken out into reasonable chunks suitable for objective performance.

### PR DESCRIPTION
**Pull request by woodbe**
This is to close #368 and close #369.

### CODE REVIEW

**Review by n-kai — COMMENTED** [review]
The term "should" is defined in CC1 as follows.

_should - within normative text, “should” indicates “that among several
possibilities one is recommended as particularly suitable, without mentioning
or excluding others, or that a certain course of action is preferred but not
necessarily required.” (ISO/IEC Directives, Part 2)._

_The CC interprets “not necessarily required” to mean that the choice of
another possibility requires a justification of why the preferred option was
not chosen._

So if we can replace "shall" with "should" in 2.3.3.4 (performance testing), but it is almost impossible for evaluators to provide "a justification of why the preferred option was not chosen" because the performance testing is the only way to estimate the FMR, FAR, FNMR and FRR in the biometric industry.

We can replace "shall" with "should" in 2.3.4.4 (quality assessment) because biometric vendors may propose different quality assessment procedure or testing. 

So I agree with changes in 2.3.4.4 but disagree with changes in 2.3.3.4.

**Comment by woodbe** [comment]
Accidental changes made to 2.3.3 (lines 427 & 430) need to be reverted and changes made properly to 2.3.2.

File: Supporting Documents/BS_SD.adoc [hunk 1]
```diff
@@ -352,7 +352,7 @@ The BMD can make a priori assumptions about the usefulness or efficacy of the cr
 --
 . The BMD shall also provide information about how authentication templates are created and, where applicable, updated. The information shall be detailed enough to conduct the EA for authentication template described below.
 . AGD guidance shall provide clear instructions for a user to enrol to the biometric system
-. Supplementary information (the quality assessment criteria report explained in <<Developer’s quality assessment criteria report of biometric samples>>) shall describe the efficacy of quality metrics, how the efficacy is tested or confirmed and how the low-quality samples can be generated
+. If supplementary information in addition to the information provided in b), (the quality assessment criteria report explained in <<Developer’s quality assessment criteria report of biometric samples>>) shall describe the efficacy of quality metrics, how the efficacy is tested or confirmed and how the low-quality samples can be generated
```
Comment (gfiumara) [inline Supporting Documents/BS_SD.adoc:355]:
```suggestion
. If supplementary information is provided in addition to the information provided in b), (the quality assessment criteria report explained in <<Developer’s quality assessment criteria report of biometric samples>>) shall describe the efficacy of quality metrics, how the efficacy is tested or confirmed and how the low-quality samples can be generated
```

File: Supporting Documents/BS_SD.adoc [hunk 2]
```diff
@@ -495,7 +495,7 @@ Following input is required from the developer.
 The BMD can make a priori assumptions about the usefulness or efficacy of the criteria or metrics. If standard quality metrics are assigned, the BMD may refer to the standard that defines quality metrics. If developer defined quality assessment is selected, the BMD shall provide the same level of information about the assessment criteria as an equivalent image quality standard for the specific modality (e.g. <<ISO29794-4, ISO/IEC 29794-4>> for fingerprint) does.
 --
 . AGD guidance shall provide clear instruction for a user to verify one's biometric
-. Supplementary information (the quality assessment criteria report explained in <<Developer’s quality assessment criteria report of biometric samples>>) shall describe the efficacy of quality metrics, how the efficacy is tested or confirmed and how the low-quality samples can be generated
+. If supplementary information in addition to the information provided in b), (the quality assessment criteria report explained in <<Developer’s quality assessment criteria report of biometric samples>>) shall describe the efficacy of quality metrics, how the efficacy is tested or confirmed and how the low-quality samples can be generated
```
Comment (gfiumara) [inline Supporting Documents/BS_SD.adoc:498]:
```suggestion
. If supplementary information is provided in addition to the information provided in b), (the quality assessment criteria report explained in <<Developer’s quality assessment criteria report of biometric samples>>) shall describe the efficacy of quality metrics, how the efficacy is tested or confirmed and how the low-quality samples can be generated
```

**Review by gfiumara — APPROVED** [review]

**Review by n-kai — APPROVED** [review]

**Pull request merged**